tests/test_other.py

```diff
@@ -8901,6 +8901,26 @@ def test_link(self):
     return 0;
 }''', '', force_c=True)
 
+  def test_link2(self):
```

Comment:
Can you give these more meanful names.  `test_linking_send` and `test_linking_recv` maybe?

Reply:
Ok